build.gradle

```diff
@@ -152,8 +152,8 @@ signing {
     required isCI
 
     useInMemoryPgpKeys(
-        providers.environmentVariable("PGP_SIGNING_KEY").forUseAtConfigurationTime().orNull,
-        providers.environmentVariable("PGP_SIGNING_KEY_PASSPHRASE").forUseAtConfigurationTime().orNull)
+        providers.environmentVariable("PGP_SIGNING_KEY").orNull,
+        providers.environmentVariable("PGP_SIGNING_KEY_PASSPHRASE").orNull)
```

Comment:
Small note: suggesting to move the `)` on the next line

Reply:
Fixed